packages/TelegramClient-UI.package/TCUChatMessageList.class/instance/addAtBottom..st

```diff
@@ -1,9 +1,13 @@
 messages
 addAtBottom: aMessage
 
-	| wrapper |	
+	| wrapper viewNewestMessage |
+
 	(self isFromCurrentChat: aMessage) ifFalse: [^ self].
+	viewNewestMessage := self vScrollBar maximumValue = self vScrollBar value.
 	
 	wrapper := self wrapperFor: (TCUMessage newFrom: aMessage).
 	self scroller addMorphBack: wrapper.
-	self items addLast: wrapper.
\ No newline at end of file
+	self items addLast: wrapper.
+	
+	viewNewestMessage ifTrue: [ self scrollToNewestMessage. ].
```

Comment:
How about renaming to "wasViewingNewestMessage"? Could also be a separate query method

Reply:
Good idea 👍